csharp/src/Ice/TransportInstance.cs

```diff
@@ -6,28 +6,28 @@
 
 namespace IceInternal
 {
-    public class ProtocolInstance
+    public class TransportInstance
     {
-        public ProtocolInstance(Ice.Communicator communicator, short type, string protocol, bool secure)
+        public TransportInstance(Ice.Communicator communicator, short type, string transport, bool secure)
         {
             Communicator = communicator;
             TraceLevel = Communicator.TraceLevels.Network;
             TraceCategory = Communicator.TraceLevels.NetworkCat;
             Logger = Communicator.Logger;
             Type = type;
-            Protocol = protocol;
+            Transport = transport;
```

Comment:
I don't like this instance suffix naming very much, I know it is a renaming of the previous schema but, shouldn't it just be a Transport and the Transport string its name.